migration-guide/redhat-to-ibm.adoc

```diff
@@ -0,0 +1,426 @@
+BOMs are ordinary `pom.xml` files — they contain no source code and their only purpose is to declare their bundled modules.
+
+.Always use the Maven Bill-of-Materials (BOM)
+[IMPORTANT]
+====
+It is highly recommended to use a bill of materials (BOM) to manage project depencencies, as it simplifies the dependency management by organizing a group of depdenencies under a single version. 
+====
+
+A bill of materails (BOM) can be added to an existing POM file by adding it to the **_dependencyManagement_** section as a dependency with a pom type:
+
+include::../partials/maven-bom-now.adoc[]
+
+When a bill of materials (BOM) is used it is no longer necessary to specify the depenency's version, as that comes from the BOM, so your depenencies end up looking like this:
+
+[source,xml]
+----
+<dependencies>
+    <dependency>
+        <groupId>org.drools</groupId>
+        <artifactId>drools-compiler</artifactId>
+    </dependency>
+</dependencies>
+----
+
+[NOTE]
+====
+If, during the course of your Maven dependency update, you encounter any dependencies that are *not* included in the bill of materials (BOM), then use the Maven Library Version noted in the table below, such as:
+====
+
+[source,xml]
+----
+<dependencies>
+    <dependency>
+        <groupId>org.drools</groupId>
+        <artifactId>drools-compiler</artifactId>
+        <version>7.67.2.Final-redhat-00006</version>
+    </dependency>
+</dependencies>
+----
+
+== Product Version Mapping
+
+{PRODUCT_LONG} v{VERSION_OLD} is essentially a blue-washed version of the former Red Hat 7.13.x offering:
+
+The following table is useful in determining which community release is bound to is section details the mapping of https://www.kie.org/[KIE Community] releases to either the former Red Hat enterprise release or the current {PRODUCT_LONG} enterprise release.
+
+include::../partials/version-mapping-now.adoc[]
+
+== Step 3: Update Tooling & Runtime
+
+Automated update tools are often provided with both patch updates and new minor versions of {PRODUCT} to facilitate updating certain components of {PRODUCT}, such as Decision/Business Central, KIE Server, and the headless controller.
+Other {PRODUCT} artifacts, such as the decision engine and standalone Decision/Business Central, are released as new artifacts with each minor release and you must reinstall them to apply the update.
+
+[NOTE]
+====
+You can deploy the latest version of {PRODUCT} as either a new installation or as a patch update from one of the previous {PRODUCT} patch releases.
+====
+
+You can use the same automated update tool to apply both patch updates and minor release upgrades to {PRODUCT} v{VERSION_OLD}.
+Patch updates of {PRODUCT}, such as an update from version 8.0 to 8.0.3, include the latest security updates and bug fixes.
+Minor release upgrades, such as an upgrade from version 8.0.x to 8.1, will include enhancements, security updates, and bug fixes.
```

Comment:
IIRC the update tool won't work in a early days combination

from 7.13.0 to 8.0.0 or something alike